plugin/rename.py

```diff
@@ -24,10 +24,70 @@
 import sublime_plugin
 
 
+BUTTONS_TEMPLATE = """
+<style>
+    html {{
+        background-color: transparent;
+        height: 2rem;
+        padding-top: 0.3rem;
```

Comment:
Can you just set `margin-top` and `margin-bottom` if you all you want is to add some extra padding?

Explicit height is bad since it implicitly depends on the content height and needs to be kept in sync.